l1-contracts/contracts/state-transition/VerifierFflonk.sol

```diff
@@ -0,0 +1,1026 @@
+uint256 internal constant PVS_ALPHA_1 = 0x264 + 0xc0;
+// final evaluation challenge
+uint256 internal constant PVS_Y = 0x264 + 0xe0;
+// convenience
+uint256 internal constant PVS_VANISHING_AT_Z = 0x264 + 0x100;
+uint256 internal constant PVS_VANISHING_AT_Z_INV = 0x264 + 0x120;
+uint256 internal constant PVS_L_0_AT_Z = 0x264 + 0x140;
+uint256 internal constant MAIN_GATE_QUOTIENT_AT_Z = 0x264 + 0x160;
+uint256 internal constant COPY_PERM_FIRST_QUOTIENT_AT_Z = 0x264 + 0x180;
+uint256 internal constant COPY_PERM_SECOND_QUOTIENT_AT_Z = 0x264 + 0x1a0;
+// ================Opening State================
+// h0, h1, h2, h2_shifted 
+uint256 internal constant OPS_OPENING_POINTS = 0x264 + 0x1c0 + 0x00; // 4 slots
+uint256 internal constant OPS_Y_POWS = 0x264 + 0x1c0 + 0x80; // 9 SLOTS
+
+// ================Pairing State================
+
+uint256 internal constant PS_VANISHING_AT_Y = 0x264 + 0x1c0 + 0x1a0;
+uint256 internal constant PS_INV_ZTS0_AT_Y = 0x264 + 0x1c0 + 0x1c0;
+uint256 internal constant PS_SET_DIFFERENCES_AT_Y = 0x264 + 0x1c0 + 0x1e0; // 3 slots
+uint256 internal constant PS_MINUS_Z = 0x264 + 0x1c0 + 0x240; // 2 slots
+uint256 internal constant PS_R_EVALS = 0x264 + 0x1c0 + 0x280; // 3 slots
+
+// ================In Memory(from Proof)================
+uint256 internal constant MEM_PROOF_PUBLIC_INPUT_SLOT = 0x264 + 0x1c0 + 0x2e0;
+
+uint256 internal constant MEM_PROOF_COMMITMENT_0_G1_X = 0x264 + 0x1c0 + 0x2e0 + 0x20;
+uint256 internal constant MEM_PROOF_COMMITMENT_0_G1_Y = 0x264 + 0x1c0 + 0x2e0 + 0x40;
+uint256 internal constant MEM_PROOF_COMMITMENT_1_G1_X = 0x264 + 0x1c0 + 0x2e0 + 0x60;
+uint256 internal constant MEM_PROOF_COMMITMENT_1_G1_Y = 0x264 + 0x1c0 + 0x2e0 + 0x80;
+uint256 internal constant MEM_PROOF_COMMITMENT_2_G1_X = 0x264 + 0x1c0 + 0x2e0 + 0xa0;
+uint256 internal constant MEM_PROOF_COMMITMENT_2_G1_Y = 0x264 + 0x1c0 + 0x2e0 + 0xc0;
+uint256 internal constant MEM_PROOF_COMMITMENT_3_G1_X = 0x264 + 0x1c0 + 0x2e0 + 0xe0;
+uint256 internal constant MEM_PROOF_COMMITMENT_3_G1_Y = 0x264 + 0x1c0 + 0x2e0 + 0x100;
+
+uint256 internal constant MEM_PROOF_EVALUATIONS = 0x264 + 0x1c0 + 0x2e0 + 0x120; // 15 slots
+
+uint256 internal constant MEM_PROOF_MONTGOMERY_LAGRANGE_BASIS_INVERSE = 0x264 + 0x1c0 + 0x2e0 + 0x120 + 0x1e0; // 1 slots
+
+uint256 internal constant MEM_LAGRANGE_BASIS_DENOMS = 0x264 + 0x1c0 + 0x2e0 + 0x120 + 0x200; //18 slots
+uint256 internal constant MEM_LAGRANGE_BASIS_DENOM_PRODUCTS = 0x264 + 0x1c0 + 0x2e0 + 0x120 + 0x440; // 18 slots
+uint256 internal constant MEM_PROOF_LAGRANGE_BASIS_EVALS = 0x264 + 0x1c0 + 0x2e0 + 0x120 + 0x680; // 18 Slots
+
+// ================Constants================
+uint256 internal constant PROOF_PUBLIC_INPUTS_LENGTH = 1;
+uint256 internal constant PROOF_LENGTH = 24;
+uint256 internal constant PROOF_EVALUATIONS_LENGTH = 15;
+uint256 internal constant TOTAL_LAGRANGE_BASIS_INVERSES_LENGTH = 18;
+
+
+/// @inheritdoc IVerifier
+function verificationKeyHash() external pure returns (bytes32 vkHash) {
```

Comment:
Why did you change this function, it was different on the audit.

The idea for this function to return commitment of verification keys. Here it just calculate hash of **constant**s, please restore the code and don't change audited code.

Reply:
I changed the function based on the suggestions during the audit (so the new change was also audited). Earlier the function used to store the values which were also constants, in memory and then produce the hash of it, here we have made those values as constants and take the hash out of it.